src/wp-includes/blocks/navigation.php

```diff
@@ -1422,9 +1418,9 @@ function block_core_navigation_update_ignore_hooked_blocks_meta( $post ) {
 	}
 }
 
-// Injection of hooked blocks into the Navigation block relies on some functions present in WP >= 6.4
-// that are not present in Gutenberg's WP 6.4 compatibility layer.
-if ( function_exists( 'get_hooked_blocks' ) ) {
+// Injection of hooked blocks into the Navigation block relies on some functions present in WP >= 6.5
+// that are not present in Gutenberg's WP 6.5 compatibility layer.
+if ( function_exists( 'get_hooked_block_markup' ) ) {
```

Comment:
@ockham, should it be moved to WordPress Core instead? It looks like it will never get executed in the Gutenberg plugin with the guard call included.